deployment/dynatrace-azure-logs.sh

```diff
@@ -235,259 +228,40 @@ while (( "$#" )); do
     esac
 done
 
-if ! $RUN_INTERACTIVE_MODE
-then
-    check_arg --deployment-name "$DEPLOYMENT_NAME" "$DEPLOYMENT_NAME_REGEX"
-    check_arg --resource-group "$RESOURCE_GROUP" ""
-    check_arg --event-hub-connection-string "$EVENT_HUB_CONNECTION_STRING" "$EVENT_HUB_CONNECTION_STRING_REGEX"
-    check_arg --event-hub-name "$EVENT_HUB_NAME" "$EVENT_HUB_NAME_REGEX"
-    if [ -n "$FILTER_CONFIG" ]; then check_arg --filter-config "$FILTER_CONFIG" "$FILTER_CONFIG_REGEX";fi
-    if [ -n "$TAGS" ]; then check_arg --tags "$TAGS" "$TAGS_REGEX"; fi
-
-    if [ -z "$USE_EXISTING_ACTIVE_GATE" ]; then USE_EXISTING_ACTIVE_GATE=false; fi
-    if [ -z "$TARGET_URL" ]
-    then
-        echo "No --target-url"
-        exit 1
-     else
-        if [[ "$USE_EXISTING_ACTIVE_GATE" == "false" ]] && ! [[ "${TARGET_URL}" =~ $DYNATRACE_TARGET_URL_REGEX ]]
-        then
-            echo -e "\e[91mERROR: \e[37mNot correct --target-url. Example of proper url for deployment with new ActiveGate: https://<your_environment_ID>.live.dynatrace.com"
-            exit 1
-        elif [[ "$USE_EXISTING_ACTIVE_GATE" == "true" ]] && ! ([[ "${TARGET_URL}" =~ $ACTIVE_GATE_TARGET_URL_REGEX ]] || [[ "${TARGET_URL}" =~ $DYNATRACE_TARGET_URL_REGEX ]])
-        then
-            echo -e "\e[91mERROR: \e[37mNot correct --target-url. Example of proper url for deployment with existing ActiveGate:"
-            echo -e "  - for Public ActiveGate: https://<your_environment_ID>.live.dynatrace.com"
-            echo -e "  - for Environment ActiveGate: https://<your_activegate_IP_or_hostname>:9999/e/<your_environment_ID>"
-            exit 1
-        fi
-    fi
-
-    if [ -z "$TARGET_API_TOKEN" ]; then echo "No --target-api-token"; exit 1; fi
-    if [[ "$USE_EXISTING_ACTIVE_GATE" == "false" ]] && [ -z "$TARGET_PAAS_TOKEN" ]; then echo "No --target-paas-token"; exit 1; fi
-    if [ -z "$REQUIRE_VALID_CERTIFICATE" ]; then REQUIRE_VALID_CERTIFICATE=false; fi
-    if [ -z "$SFM_ENABLED" ]; then SFM_ENABLED=false; fi
-    if [[ "$USE_EXISTING_ACTIVE_GATE" == true ]]; then DEPLOY_ACTIVEGATE=false;else DEPLOY_ACTIVEGATE=true;fi
-    if [ -z "$REPOSITORY_RELEASE_URL" ]; then REPOSITORY_RELEASE_URL=${FUNCTION_REPOSITORY_RELEASE_URL}; fi
-    print_all_parameters
+check_arg --deployment-name "$DEPLOYMENT_NAME" "$DEPLOYMENT_NAME_REGEX"
```

Comment:
Consider adding to check if az is installed. This was already done for Interactive mode but omitted in non-interactive mode.
```
    echo -e "\033[1;34mDynatrace function for Azure logs ingest"
    echo -e "\033[0;37m"
    if ! command -v az &> /dev/null
    then

        echo -e "\e[91mERROR: \e[37mAzure CLI is required to install Dynatrace function. It should be already installed in Cloud Shell."
        echo -e "If you are running this script from other hosts go to following link in your browser and install latest version of Azure CLI:"
        echo -e
        echo -e "https://docs.microsoft.com/en-us/cli/azure/install-azure-cli"
        echo -e
        echo
        exit
    fi
```